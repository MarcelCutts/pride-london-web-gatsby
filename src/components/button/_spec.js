import React from 'react'
import { shallow } from 'enzyme'
import Button from './'

describe('Button component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Button />)

    expect(wrapper).toMatchSnapshot()
  })

  it('should have property of colour xxx if primary', () => {})

  it('should fire correct method when set onClick property', () => {})

  it('should not be clickable if disabled', () => {})

  it('should match the corresponding sizes if small and wide are passed in as props', () => {})

  it('should redirect the page if button is link and to is defined', () => {})
})
