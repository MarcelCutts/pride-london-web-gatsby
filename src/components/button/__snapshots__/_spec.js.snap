// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should match snapshot 1`] = `
<styled.button
  disabled={false}
  href={null}
  onClick={[Function]}
  primary={false}
  small={false}
  type="button"
  wide={true}
>
  Button
</styled.button>
`;
